docroot/modules/custom/mass_validation/mass_validation.module

```diff
@@ -207,17 +211,24 @@ function mass_validation_form_node_form_alter(&$form, FormStateInterface $form_s
 function _mass_validation_field_organizations_validate(array &$form, FormStateInterface $form_state) {
   if ($form_state->hasValue('field_organizations')) {
     if (isset($form['field_organizations']['widget'][0]['target_id']) && $form['field_organizations']['widget'][0]['target_id']['#required'] == TRUE) {
-      // field_organizations is a multi value field, on which we get a false error if one or more
-      // values are present, but the first value is missing. If such a false error is present then
-      // we remove that error, but we keep other genuine errors connected to the form submission.
-      // The way we do this is inspired by code in the conditional_fields module.
+      // field_organizations is a multi value field,
+      // on which we get a false error if one or more
+      // values are present, but the first value is missing.
+      // If such a false error is present then
+      // we remove that error, but we keep other genuine errors
+      // connected to the form submission.
+      // The way we do this is inspired by code in
+      // the conditional_fields module.
```

Comment:
This change and several others are not desireable IMO. The 80 char limit is from old days of teletype. Editors can line wrap since about 1985. If your editor is reformatting unrelated code for line length, please disable that. Its OK for this PR.